.github/workflows/main.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       - static-analysis
       - unit-tests-with-coverage
       - integration-test
-    if: ${{ github.ref_name == 'main' }}
+#    if: ${{ github.ref_name == 'main' }}
```

Comment:
Why is this commented out?

Reply:
I was testing the new workflow. I'll leave this comment open to make sure I don't forget to un-comment it.

Reply:
Removed